app/main/index.js

```diff
@@ -34,7 +35,15 @@ const mainURL = 'file://' + path.join(__dirname, '../renderer', 'main.html');
 
 const singleInstanceLock = app.requestSingleInstanceLock();
 if (singleInstanceLock) {
-	app.on('second-instance', () => {
+	app.on('second-instance', (event, argv) => {
+		// uri scheme handler for windows
+		if (process.platform === 'win32') {
+			deepLinkingUrl = argv.slice(1);
+			if (mainWindow) {
+				mainWindow.webContents.send('deep-linking-url', deepLinkingUrl[isDev ? 1 : 0]);
```

Comment:
Can you explain what's going on here?

Reply:
Right, so we get the URL passed to the app in the `argv` object and then we need to send that to the renderer process. 
I noticed that the URL is present at the first element of the array when I build the app and it was present at the second element when in development mode. Not sure why, but I can look around if required.